examples/3-mutation/src/Grid.re

```diff
@@ -1,17 +1,17 @@
 open ReasonUrql;
 open Client;
-
-type dog = {
-  key: string,
-  name: string,
-  breed: string,
-  likes: int,
-  pats: int,
-  treats: int,
-  bellyscratches: int,
-  description: string,
-  imageUrl: string,
-};
+open Types;
```

Comment:
I'd prefer to keep the `type dog` local here just since it's used for the `@bsDecoder` stuff from `graphql_ppx` and it keeps it all on one place.

Reply:
It was just to test it. Removed.